test/test_misc_utils.py

```diff
@@ -3159,6 +3159,16 @@ def test_dict_to_keys_and_values():
     assert dict_to_keys_and_values({}) == []
 
 
+def test_is_c4_arn():
+    assert is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-cgap-production-stack-Productionblue-fzOm82DGZKGZ')
+    assert is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-cgap-production-stack-CgapProductionblue-fzOm82DGZKGZ')
+    assert is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-fourfront-hotseat-stack-Hotseat-ZnIAAAz8Apxf')
+    assert is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-fourfront-production-stack-FourfrontProd-hU0JiwOeWfdm')
+    assert not is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-abc-webdev-stack-CgapWebdev-rSLwZBbdVTtx')
+    assert not is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/default')
+    assert not is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-def-mastertest-stack-FourfrontMaster-O25WVcRMpExr')
```

Comment:
Probably picking some completely different ARN is worth testing, too, though I guess /default thing above is not really that different. I'd put that test case last if you make the c4- things return true, just for grouping.